front/app/components/UI/Error/index.tsx

```diff
@@ -179,32 +172,33 @@ export type TFieldName =
   | 'category_name'
   | 'nav_bar_item_title_multiloc';
 
-export default class Error extends PureComponent<Props, State> {
-  static defaultProps: DefaultProps = {
-    marginTop: '3px',
-    marginBottom: '0px',
-    showIcon: true,
-    showBackground: true,
-    className: '',
-    animate: true,
-  };
-
-  constructor(props) {
-    super(props);
-    this.state = {
-      mounted: false,
-    };
-  }
-
-  componentDidMount() {
-    this.setState({ mounted: true });
-  }
-
-  componentWillUnmount() {
-    this.setState({ mounted: false });
-  }
+const Error = (props: Props) => {
+  const containerRef = useRef<HTMLDivElement | null>(null);
+
+  const {
+    text,
+    apiErrors,
+    fieldName,
+    marginTop = '3px',
+    marginBottom = '0px',
+    showIcon = true,
+    showBackground = true,
+    className = '',
+    animate = true,
+    id,
+  } = props;
+
+  useEffect(() => {
+    if (text || apiErrors) {
```

Comment:
Why do we scroll it into view where there is text?

Reply:
Because I imagine if the error comes from a front-end validation, for example (not an api error), and is passed to the component via the `text` prop, we still want to scroll to it to highlight it.